### PR TITLE
Replacing nohup command with setsid

### LINKED ISSUE #44: Replace nohup command with setsid

### PR DESCRIPTION
**Pull request by OmerCS8**
Replacing nohup command with setsid command in setup.sh (as Barak recommended)

Dependent on #43 
fix #44

### CODE REVIEW

**Review by yulisuliman — DISMISSED** [review]

**Review by MatanP12 — DISMISSED** [review]
LGTM

File: setup.sh [hunk 1]
```diff
@@ -10,7 +10,7 @@ cd /vagrant
 # Install dependencies with Pipenv
 pipenv sync --dev
 
-# run our app. Nohup "&" are used to let the setup script finish
-# while our app stays up. The app logs will be collected in nohup.out
-nohup pipenv run python manage.py runserver 0.0.0.0:8000 &
-
+# run our app. setsit, the parentheses and "&" are used to perform a "double
+# fork" so that out app stays up after the setup script finishes.
```
Comment (natifridman) [inline setup.sh:14]:
```suggestion
# fork" so that our app stays up after the setup script finishes.
```

**Review by natifridman — CHANGES_REQUESTED** [review]

**Review by TomerNewmanPrograms — DISMISSED** [review]
lgtm

**Review by AlonShakaroff — DISMISSED** [review]
LGTM

**Review by natifridman — APPROVED** [review]

**Review by MatanP12 — APPROVED** [review]

**Review by yulisuliman — APPROVED** [review]

**Comment by OmerCS8** [comment]
> Please squash your commits

Done

**Review by TomerNewmanPrograms — APPROVED** [review]
lgtm

**Review by AlonShakaroff — APPROVED** [review]
LGTM

**Pull request merged**